### PR TITLE
Add a blog post announcing NVIDIA sponsorship

### PR DESCRIPTION
**Pull request by francoisgoupil**
This blog post is aiming to announce Nvidia sponsorship to the scikit-learn consortium as well as showing the broader support of Nvidia to the community project. 

@betatim @mmccarty

### CODE REVIEW

File: _posts/2023-07-11-nvidia-is-a-new-sponsor.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Our partnership with the scikit-learn consortium will also enable us to collaborate more closely with the scikit-learn community, and provide us with insights into how we can improve NVIDIA’s [RAPIDS open-source libraries](https://developer.nvidia.com/rapids) to better serve their needs. We are committed to working with the foundation to ensure that scikit-learn remains a powerful and easy to use machine learning library that meets the needs of data science practitioners in science and industry.
+
+NVIDIA’s commitment to scikit-learn goes beyond financial support. We have hired [Tim Head](), an experienced open-source maintainer, to work full-time on the project. This is not Tim’s first open-source rodeo. He has previously contributed to several high-profile open-source projects, including Project Jupyter. His focus will be reviewing pull requests and coordinating the development of large features. Tim was recently elected as a core maintainer of scikit-learn. His expertise and experience will be invaluable in ensuring the continued growth and success of the project.
```
Comment (betatim) [inline _posts/2023-07-11-nvidia-is-a-new-sponsor.md:42]:
```suggestion
NVIDIA’s commitment to scikit-learn goes beyond financial support. We have hired [Tim Head](https://betatim.github.io), an experienced open-source maintainer, to work full-time on the project. This is not Tim’s first open-source rodeo. He has previously contributed to several high-profile open-source projects, including Project Jupyter. His focus will be reviewing pull requests and coordinating the development of large features. Tim was recently elected as a core maintainer of scikit-learn. His expertise and experience will be invaluable in ensuring the continued growth and success of the project.
```

**Comment by betatim** [comment]
Added a comment that adds a link to my website to my name. Otherwise looks good to me.

File: _posts/2023-07-11-nvidia-is-a-new-sponsor.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    image: "francois_goupil.jpeg"
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+<span style="color:red">*Sponsored blog post* </span>
+
+We are thrilled to announce that [NVIDIA](https://www.nvidia.com) has joined the [scikit-learn consortium](https://scikit-learn.fondation-inria.fr/) as a corporate partner. As a leading provider of GPU-accelerated computing solutions, we recognize the importance of machine learning and the role it plays in the growth of many industries and areas of science. Our partnership with the scikit-learn consortium demonstrates our commitment to supporting the development and advancement of open-source software in the machine learning community.
```
Comment (mmccarty) [inline _posts/2023-07-11-nvidia-is-a-new-sponsor.md:28]:
```suggestion
We are thrilled to announce that [NVIDIA](https://www.nvidia.com) has joined the [scikit-learn consortium](https://scikit-learn.fondation-inria.fr/) as a corporate partner. As a leading provider of GPU-accelerated computing solutions, we at NVIDIA recognize the importance of machine learning and the role it plays in the growth of many industries and areas of science. Our partnership with the scikit-learn consortium demonstrates our commitment to supporting the development and advancement of open-source software in the machine learning community.
```

**Review by mmccarty — APPROVED** [review]

**Comment by reshamas** [comment]
@francoisgoupil ready to merge?

File: _posts/2023-07-11-nvidia-is-a-new-sponsor.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+categories:
+  - Funding
+tags:
+  - Sponsor
+
+featured-image: NVIDIAxsklearn.jpg
+
+postauthors:
+  - name: NVIDIA
+    website: https://nvidia.com/
```
Comment (fredoh1) [inline _posts/2023-07-11-nvidia-is-a-new-sponsor.md:14]: Please change to "https://developer.nvidia.com/gpu-accelerated-libraries"

File: _posts/2023-07-11-nvidia-is-a-new-sponsor.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    image: "francois_goupil.jpeg"
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+<span style="color:red">*Sponsored blog post* </span>
+
+We are thrilled to announce that [NVIDIA](https://www.nvidia.com) has joined the [scikit-learn consortium](https://scikit-learn.fondation-inria.fr/) as a corporate partner. As a leading provider of GPU-accelerated computing solutions, we at NVIDIA recognize the importance of machine learning and the role it plays in the growth of many industries and areas of science. Our partnership with the scikit-learn consortium demonstrates our commitment to supporting the development and advancement of open-source software in the machine learning community.
```
Comment (fredoh1) [inline _posts/2023-07-11-nvidia-is-a-new-sponsor.md:28]: Please you add this (https://developer.nvidia.com/open-source) hyper link to "open-source software" statement.

Comment (betatim) [inline _posts/2023-07-11-nvidia-is-a-new-sponsor.md:28]:
```suggestion
We are thrilled to announce that [NVIDIA](https://www.nvidia.com) has joined the [scikit-learn consortium](https://scikit-learn.fondation-inria.fr/) as a corporate partner. As a leading provider of GPU-accelerated computing solutions, we at NVIDIA recognize the importance of machine learning and the role it plays in the growth of many industries and areas of science. Our partnership with the scikit-learn consortium demonstrates our commitment to supporting the [development and advancement of open-source software](https://developer.nvidia.com/open-source) in the machine learning community.
```

Comment (betatim) [reply _posts/2023-07-11-nvidia-is-a-new-sponsor.md:28]: @francoisgoupil what do you think?

File: _posts/2023-07-11-nvidia-is-a-new-sponsor.md (hunk 3 above)
Comment (betatim) [reply _posts/2023-07-11-nvidia-is-a-new-sponsor.md:14]: I think as a general link to "NVIDIA the company as a whole" nvidia.com is a good URL to use. In addition https://developer.nvidia.com/gpu-accelerated-libraries doesn't refer to RAPIDS where the work is happening/I'm part of.

File: _posts/2023-07-11-nvidia-is-a-new-sponsor.md (hunk 4 above)
Comment (francoisgoupil) [reply _posts/2023-07-11-nvidia-is-a-new-sponsor.md:28]: I am ok with adding this link.

**Pull request merged**